sqs/provider.go

```diff
@@ -125,6 +131,32 @@ func (s *Provider) Deprovision(ctx context.Context, deprovisionData providerifac
 }
 
 func (s *Provider) Bind(ctx context.Context, bindData provideriface.BindData) (*domain.Binding, error) {
+	bindParameters := BindParameters{}
+	if len(bindData.Details.RawParameters) > 0 {
+		decoder := json.NewDecoder(bytes.NewReader(bindData.Details.RawParameters))
+		decoder.DisallowUnknownFields()
+		if err := decoder.Decode(&bindParameters); err != nil {
+			var errStr error
+			switch castErr := err.(type) {
+				case *json.UnmarshalTypeError:
+					errStr = fmt.Errorf(
+						"Expected %#v field of bind parameters to be of type %v",
+						castErr.Field,
+						castErr.Type.String(),
+					)
+				default:
+					errStr = castErr
+			}
+			// wat
+
+			return nil, brokerapi.NewFailureResponse(
```

Comment:
fix: this package is deprecated ... please switch to apiresponses.NewFailureResponse:

https://godoc.org/github.com/pivotal-cf/brokerapi/domain/apiresponses#NewFailureResponse